buildSrc/src/main/kotlin/com/ibm/wala/gradle/maven-eclipse-jsdt.gradle.kts

```diff
@@ -0,0 +1,9 @@
+/** Define a Maven repository containing Eclipse JSDT packages. */
+package com.ibm.wala.gradle
+
+repositories {
+  maven {
+    url = uri("https://artifacts.alfresco.com/nexus/content/repositories/public/")
```

Comment:
FWIW I think we stopped trying to build this project under Gradle since The Alfresco Maven repo was flaky; see #944.  Not sure if this changed recently.  If it hasn't, maybe there is some further cleanup possible?  I'm fine with the further cleanup coming in a follow-up PR

Reply:
Sure, perhaps we can simplify. I'll look into that in a separate PR to follow the current one.

Reply:
#1188 documents my intention to explore @msridhar's suggestion soon.